src/ui/processTree/darwinProcessTree.ts

```diff
@@ -43,7 +43,7 @@ export class DarwinProcessTree extends BaseProcessTree {
    * @inheritdoc
    */
   protected createProcess() {
-    return this.spawn('/bin/ps', [
+    return this.spawn('ps', [
```

Comment:
I'm aware this is the darwin file, but I figured we might as well be consistent.